woocommerce/woocommerce-ios
# Product Details: Pricing & Inventory - Mark II

Closes #857.

This PR completes the pricing & inventory section of the Product Details screen.

**Simple product / Simple product - variation, with Sale Price and SKU**
(At this point in the implementation, nothing is different about the screens)

<img src="https://user-images.githubusercontent.com/1062444/58133332-3c410000-7be9-11e9-9918-2431b10cedf4.png" width="350" />

**Simple product - variation, no sale and no SKU**
<img src="https://user-images.githubusercontent.com/1062444/58133555-c12c1980-7be9-11e9-9f0e-3437d7b0760b.png" width="350" />

**Simple product - manage stock = true**
<img src="https://user-images.githubusercontent.com/1062444/58133381-524ec080-7be9-11e9-9e07-f9bec2b1e79b.png" width="350" />

**Affiliate product**
<img src="https://user-images.githubusercontent.com/1062444/58133420-60044600-7be9-11e9-89e7-d7cf00ede96d.png" width="350" />

**Grouped product**
<img src="https://user-images.githubusercontent.com/1062444/58133434-6d213500-7be9-11e9-94dc-013d202c1cc7.png" width="350" />

## To test
1. Check out this branch and run it on the simulator
2. On your store's website, create a new order manually. Add a Grouped Product, an Affiliate Product, and a Simple product (any kind of simple product). Click "Create" to place the order.
3. Back in the app, pull-to-refresh on the Orders list. Select the order, select "Details" to see the Product list.
4. Select a row. Each product should have the corresponding rows for the Pricing & Inventory sections listed in the [Figma file](https://www.figma.com/file/CaEnarFYNDXPMR0rdpPxhPXF/Product-Details?node-id=56%3A0).

Update release notes:
